glustercli/cmd/root.go

```diff
@@ -84,14 +85,15 @@ var (
 	// set by command line flags
 	flagXMLOutput  bool
 	flagJSONOutput bool
-	flagEndpoints  []string
-	flagCacert     string
 	flagInsecure   bool
-	flagLogLevel   string
 	verbose        bool
+	flagCacert     string
+	flagLogLevel   string
 	flagUser       string
 	flagSecret     string
 	flagSecretFile string
+	flagEndpoints  []string
+	flagTimeout    int
```

Comment:
question:
can we declare ```flagTimeout``` as ```uint```?

Reply:
I initially thought about it but decided not do it though. Let me try it out and see if that can be a substitute for validating it.

Reply:
I've changed it to uint :+1: